python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4540,9 +4539,6 @@ def test_empty_df_astype(dtype, args):
         ),
         pytest.param("other", marks=pytest.mark.xfail(raises=ValueError)),
         "ignore",
-        pytest.param(
-            "warn", marks=pytest.mark.filterwarnings("ignore:Traceback")
-        ),
     ],
 )
 def test_series_astype_error_handling(errors):
```

Comment:
This test shouldn't use `xfail` for `"raise"`. We should ensure an error is raised with `pytest.raises` instead of just permitting the test to fail. This can be done in a separate PR if desired, since this PR is strictly removing deprecated features.

Reply:
Agreed, I'll keep this in mind for future work.

Reply:
Also... why is this test (of Series) in test_dataframe.py...